app/src/main/java/com/zulip/android/filters/NarrowFilterByDate.java

```diff
@@ -72,7 +73,7 @@ public String getTitle() {
 
     @Override
     public String getSubtitle() {
-        SimpleDateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy");
```

Comment:
Because `./gradlew lintDebug` was giving warning. Now I have removed it. Will be added once https://github.com/zulip/zulip-android/pull/311/ gets merged. :)